rnd/autogpt_server/autogpt_server/cli.py

```diff
@@ -0,0 +1,97 @@
+from multiprocessing import freeze_support
+from multiprocessing.spawn import freeze_support as freeze_support_spawn
+
+import click
+
+
+@click.group()
+def main():
+    """AutoGPT Server CLI Tool"""
+    pass
+
+
+@main.command()
+def background() -> None:
+    from autogpt_server.app import background_process
+
+    background_process()
+
+
+@main.command()
+def run():
+    import os
+    import pathlib
+    import subprocess
+
+    # Define the path for the new directory and file
+    home_dir = pathlib.Path.home()
+    new_dir = home_dir / ".config" / "agpt"
+    file_path = new_dir / "running.tmp"
+
+    # Create the directory if it does not exist
+    os.makedirs(new_dir, exist_ok=True)
+    if file_path.exists():
+        print("Server is already running")
+        return
+
+    sp = subprocess.Popen(
+        ["poetry", "run", "python", "autogpt_server/cli.py", "background"],
+        stdout=subprocess.DEVNULL,  # Redirect standard output to devnull
```

Comment:
should this go to a log file ?

Reply:
I've piped this to null to avoid cluttering the console with messages from a background process.

The background process, will be configured to log to file so keep access to the logs.